lib/cli/line-processor.js

```diff
@@ -68,11 +68,11 @@ module.exports = class LineProcessor {
     const [host, portString] = utils.split(authority, ':', 2, true);
     if (!host) {
       return callback(reportMissingArgument('Host'));
-    } else if (!portString) {
+    } else if (scheme !== 'ipc' && !portString) {
```

Comment:
@lundibundi, it is not possible, because then `port` will be declared inside this block (it is on line 74) and won't be visible as an argument to `commandProcessor.connect` function.
Can you explain your idea, please, because I don't understand how you want this to look like?